<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Equipements</class>
 <widget class="QDialog" name="Equipements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>711</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Equipement_Ajout">
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>351</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="Equipement_Ajout_Nom"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="Equipement_Ajout_Submit">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Adresse_Label">
        <property name="text">
         <string>Quantite</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="Nom_Label">
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="Equipement_Ajout_Cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="Equipement_Ajout_Image">
        <property name="text">
         <string>Selectionner Image Equipement</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Image</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="Equipement_Ajout_Quantite"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="Equipement_Ajout_Image_Display">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>190</y>
       <width>251</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Equipements_Affichage">
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>501</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>380</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </widget>
    <widget class="QLabel" name="Equipement_Affichage_Logo">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="Background_Pic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Background_Pic</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
